partitioner/src/main/java/io/confluent/connect/storage/partitioner/FieldPartitioner.java

```diff
@@ -16,6 +16,8 @@
 
 package io.confluent.connect.storage.partitioner;
 
+import io.confluent.connect.storage.common.StorageCommonConfig;
```

Comment:
nit: noise due to code style. You might want to skip these changes to reduce the chance of future merge conflicts.